### PR TITLE
Fix #141: Replaced broken and old links in README.md

### LINKED ISSUE #141: [Bug] Fix dead links in README.md
## Describe the bug

Some links in the [README.md](README.md) are either broken or redirect to the old GitLab repository.

## To Reproduce

1. Open README.md
2. Click on the links under [**How to contribute**](https://github.com/EOS-uiux-Solutions/user-story) section
3. Some links will redirect to a 404 page, or to the old GitLab repository

## Expected behavior

The links should redirect to their corresponding target.

## Logs

NA

## Environment (please complete the following information):

- OS: Windows 10
- Browser: Chrome

## Additional context

NA

### PR DESCRIPTION
**Pull request by kailash360**
**Issue Number**

fixes #141 

**Describe the changes you've made**

I have removed the broken links from [README.md](https://github.com/EOS-uiux-Solutions/user-story/blob/main/README.md) and replaced them with working links. I have also substituted the old GitLab links with the corresponding GitHub links.

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

NA

**Additional context (OPTIONAL)**

NA

**Test plan (OPTIONAL)**

NA

**Checklist**

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

**Provide a Deployed link of route/page that needs to review**

NA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,25 +57,25 @@ See the section about [deployment](https://facebook.github.io/create-react-app/d
...
-2. Make sure you follow our best practices: [refer to our Wiki](https://gitlab.com/SUSE-UIUX/eos/-/wikis/home). You'll find information on writing code, how to name a branch, how we release, etc.
-3. Join Slack [optional] to get in touch with the maintainers if you have any doubt: [join slack](http://slack.eosdesignsystem.com/)
-4. Make sure you fork the project, cloning it will not give you the right access to open a PR/MR. [How to open a PR in open source](https://gitlab.com/SUSE-UIUX/eos/-/wikis/Basic-git-instructions-for-beginners)
+2. Make sure you follow our best practices: [refer to our Wiki](https://github.com/EOS-uiux-Solutions/wiki/blob/main/README.md). You'll find information on writing code, how to name a branch, how we release, etc.
+3. Join Slack [optional] to get in touch with the maintainers if you have any doubt: [join slack](https://eos-community.slack.com)
+4. Make sure you fork the project, cloning it will not give you the right access to open a PR/MR. [How to open a PR in open source](https://github.com/EOS-uiux-Solutions/wiki/blob/main/pull-requests,-approvals,-and-merging-to-master.md#opening-a-pull-request-pr)
```
Comment (cyntss) [inline README.md:62]: the correct link is the same as before, but from our new Github wiki:

https://github.com/EOS-uiux-Solutions/wiki/blob/main/Basic-git-instructions-for-beginners.md

File: README.md [hunk 2]
```diff
@@ -57,25 +57,25 @@ See the section about [deployment](https://facebook.github.io/create-react-app/d
...
 
 ### Design contributions
 
 If you'd like to contribute with design changes, you'll have to do as follows:
 
 1. [Open an issue](https://github.com/EOS-uiux-Solutions/user-story/issues/new)
 2. Add all relevant information to the issue. Please be as descriptive as possible. Add links to references, images, videos, etc.
-3. While working on your design, please make sure you follow our [design guidelines](https://gitlab.com/SUSE-UIUX/eos/wikis/Design-process-and-feedback-gathering).
+3. While working on your design, please make sure you follow our [design guidelines](https://github.com/EOS-uiux-Solutions/wiki/blob/main/README.md#designing-the-eos-project).
```
Comment (cyntss) [inline README.md:70]: same as before, but from new repo

https://github.com/EOS-uiux-Solutions/wiki/blob/main/Design-process-and-feedback-gathering.md

File: README.md [hunk 3]
```diff
@@ -57,25 +57,25 @@ See the section about [deployment](https://facebook.github.io/create-react-app/d
...
 4. Once the issue is approved by us, it'll be ready to be implemented.
 
 # Learn more about the EOS Design System
 
-- [EOS Design System](https://www.eosdesignsystem.com/)
+- [EOS Design System](https://www.eosdesignsystem.com)
 
-- [EOS Icons](icons.eosdesignsystem.com/)
+- [EOS Icons](eos-icons.com)
```
Comment (cyntss) [inline README.md:77]: let's move this line to the top of the list, and let's add the protocol too (https://)

File: README.md [hunk 4]
```diff
@@ -57,25 +57,25 @@ See the section about [deployment](https://facebook.github.io/create-react-app/d
...
 1. [Open an issue](https://github.com/EOS-uiux-Solutions/user-story/issues/new)
 2. Add all relevant information to the issue. Please be as descriptive as possible. Add links to references, images, videos, etc.
-3. While working on your design, please make sure you follow our [design guidelines](https://gitlab.com/SUSE-UIUX/eos/wikis/Design-process-and-feedback-gathering).
+3. While working on your design, please make sure you follow our [design guidelines](https://github.com/EOS-uiux-Solutions/wiki/blob/main/README.md#designing-the-eos-project).
 4. Once the issue is approved by us, it'll be ready to be implemented.
 
 # Learn more about the EOS Design System
 
-- [EOS Design System](https://www.eosdesignsystem.com/)
+- [EOS Design System](https://www.eosdesignsystem.com)
```
Comment (cyntss) [inline README.md:75]: Can you move this one to the bottom and also add in second place the link to user Story?https://userstory.eosdesignsystem.com/

thank you!

**Review by cyntss — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -57,25 +57,25 @@ See the section about [deployment](https://facebook.github.io/create-react-app/d
 ### Code contributions
 
 1. Open a new issue or pick an open issue from the issue list and claim it in the comments. Make sure that the issue is confirmed so you don't work on something that will not be approved to be merged.
-2. Make sure you follow our best practices: [refer to our Wiki](https://gitlab.com/SUSE-UIUX/eos/-/wikis/home). You'll find information on writing code, how to name a branch, how we release, etc.
-3. Join Slack [optional] to get in touch with the maintainers if you have any doubt: [join slack](http://slack.eosdesignsystem.com/)
-4. Make sure you fork the project, cloning it will not give you the right access to open a PR/MR. [How to open a PR in open source](https://gitlab.com/SUSE-UIUX/eos/-/wikis/Basic-git-instructions-for-beginners)
+2. Make sure you follow our best practices: [refer to our Wiki](https://github.com/EOS-uiux-Solutions/wiki/blob/main/README.md). You'll find information on writing code, how to name a branch, how we release, etc.
+3. Join Slack [optional] to get in touch with the maintainers if you have any doubt: [join slack](https://eos-community.slack.com)
```
[the remark this answers is not in the document]
Comment (kailash360) [reply README.md]: Alright, I have fixed this in the latest commit

File: README.md (hunk 1 above)
Comment (kailash360) [reply README.md]: Fixed this in the latest commit

File: README.md (hunk 3 above)
Comment (kailash360) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (kailash360) [reply README.md]: Done

**Review by cyntss — APPROVED** [review]

**Comment by cyntss** [comment]
Thank you @kailash360 . Merging now!

**Pull request merged**